util/chunk/chunk.go

```diff
@@ -407,7 +407,16 @@ func (c *column) appendString(str string) {
 }
 
 func (c *column) appendBytes(b []byte) {
-	c.data = append(c.data, b...)
+	m := len(c.data)
+	n := m + len(b)
+	if n > cap(c.data) { // if necessary, reallocate
+		// allocate 128 times larger than what's needed, for futher growth
```

Comment:
Why is it 128 times?

Reply:
tried several value, this one can reduce ops.

Reply:
2 times is more common...

Reply:
For the current implementation, we put all the bytes in a column into a continuous memory buffer, witch is `c.data`, this may lead to a huge continuous memory allocation in some situation.

I think 2 is more safe. And when condition `n > cap(c.data)` is satisfied, `cap(c.data) == len(c.data)` is usually satisfied as well. So this optimization maybe unnecessary.